kernel/kernel-api/src/test/scala/io/delta/kernel/internal/util/SchemaUtilsSuite.scala

```diff
@@ -495,16 +496,41 @@ class SchemaUtilsSuite extends AnyFunSuite {
     assert(e.getMessage == "Cannot validate updated schema when column mapping is disabled")
   }
 
+  private val primitiveSchema = new StructType()
```

Comment:
Cleaning up duplication in the tests by moving some of the constant schemas out to variables.